CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 Details changes in each release of EarlGrey. EarlGrey follows
 [semantic versioning](http://semver.org/).
 
+## [1.6.2](https://github.com/google/EarlGrey/tree/1.6.2) (01/06/2017)
+
+```
+Baseline: [0cdda9c]
++ [0cdda9c]: EarlGrey Sync for 1.6.2
+```
+
+### Bug Fixes
+* Fixed EarlGrey.swift for Swift 3.0
+* Improved web tests to work with current google.com UI.
+* Fixed a bug in the visibility checker for 32bit platform.
```

Comment:
Also,

* Fixed flakiness caused by NSDate issues in EarlGreyExampleSwiftTests.